underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -121,15 +124,21 @@ public void close() throws IOException {
   @Override
   public FSDataOutputStream create(String path) throws IOException {
     IOException te = null;
-    int cnt = 0;
-    while (cnt < MAX_TRY) {
+    RetryPolicy retryPolicy = new SleepingRetry(MAX_TRY) {
```

Comment:
Makes sense but do u think if CountingRetry should extend SleepingRetry with 0 sleep secs to like ExponentialBackoffRetry which is similar to what this code does except naming or do u think a separate implementation is needed?